complex.c

```diff
@@ -2407,9 +2429,13 @@ float_arg(VALUE self)
  * == Polar Coordinates
  *
  * The polar coordinates of a complex number
- * are called the _absolute_ and _argument_ parts;
+ * are called the _absolute_ and _argument_ parts.
```

Comment:
Was there a reason why this semicolon was replaced with a period?

Reply:
Fixed.